logback-core/src/test/java/ch/qos/logback/core/rolling/TimeBasedRollingWithArchiveRemoval_Test.java

```diff
@@ -280,17 +292,17 @@ void buildRollingFileAppender(long currentTime, String fileNamePattern, int maxH
     rfa.start();
   }
 
-  boolean DO_CLEAN_HISTORY_ON_START = true;
-  boolean DO_NOT_CLEAN_HISTORY_ON_START = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/tony19/logback/file/2371885452/issues/source?bid=3222618&fileBranchId=3222682#l295)